KomaMRICore/src/rawdata/ISMRMRD.jl

```diff
@@ -1,3 +1,12 @@
+"""
+It is the same as defined in MRIFiles without the need to import MRIFiles
+"""
+struct Limit
+    minimum::Int
+    maximum::Int
+    center::Int
+end
+
```

Comment:
Not sure about this, now there will be two definitions `KomaMRICore.Limit` and `MRIReco.Limit`, not  sure what the writter will do.

Reply:
I can confirm that this does not write the MRD correctly 

![image](https://github.com/cncastillo/KomaMRI.jl/assets/5957134/a0d7b4bb-e140-4b67-b28e-d7b7c830c8c1)

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/231/commits/f42277c2cc049899f3ed25880599e7881d18e321.

The problem is we don't want to use MRIFiles in KomaMRICore, but Limit() is part of MRIFiles.
I created a new function to update the parameters of a raw::RawAcquisitionData variable.
A user must update these parameters before saving a raw variable to a .mrd.